### PR TITLE
DOCSP-23576 enableUserWriteBlock requirements

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23576-uri-actiontypes-v0.9.0/reference/api/start/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-23576)

### CODE REVIEW

File: source/reference/api/start.txt [hunk 1]
```diff
@@ -25,6 +25,8 @@ Requirement
 To use the ``start`` endpoint, ``mongosync`` must be in the ``IDLE``
 state.
 
+.. include:: /includes/fact-write-blocking-requirement.rst
```
Comment (mdb-ashley) [inline source/reference/api/start.txt:28]: [suggestion]

Should this section be titled "Requirements" since there are two requirements?

**Review by mdb-ashley — APPROVED** [review]
LGTM, pending your thoughts about a single suggestion. Nice work with the single-sourcing here. Thanks for handling this work, Dave!

File: source/reference/api/start.txt (hunk 1 above)
Comment (davemungo) [reply source/reference/api/start.txt:28]: Done

**Review by davemungo — COMMENTED** [review]
@mdb-ashley thanks for your feedback, sending on to external review next.

**Pull request merged**